modules/special-rules/nav.adoc

```diff
@@ -18,43 +18,60 @@
   *** xref:vortex-missile.adoc[]
 
  ** xref:special-abilities.adoc[]
-  *** xref:anti-personnel-ability.adoc[]
-  *** xref:artillery.adoc[]
+
+ ** Special abilities: unit design modifiers
```

Comment:
```suggestion
 ** Unit data sheet modifiers
```
...but see also my comment on the army abilities.

Reply:
Hmm. How about this?

```suggestion
 ** Data sheet modifiers
```

Reply:
Then we have:

Special abilities
- Data sheet modifiers
- Unit abilities
- Detachment abilities
- Army rules

Reply:
> As an aside, I wonder if we can come up with a good collective name for these four types of 'ability' rules, for the sake of design discussions?

What issues are there with 'special abilities'?

Reply:
> My mind was moving the same way on the others too. The only one I'm slightly ambivalent about is Army rules because it feels too general. I'll add to the comments on that one separately.

How about Faction rules?